app/webpack/components/authentication/sign_in.vue

```diff
@@ -251,7 +251,10 @@ export default {
 
       if (data.user.email_verified === false) {
         const user = firebase.auth().currentUser;
-        return user.sendEmailVerification().then(() => {
+        const actionCodeSettings = {
+          url: 'https://soulmedicine.io'
```

Comment:
Unfortunately this won't work for the preview, staging sites, and any review apps.

Reply:
We have a `SITE_BASE_URL` which you can pass into the Vue component from the Rails view.

Reply:
Note though what we do for the mailer: https://github.com/chaynHQ/soulmedicine/blob/master/config/initializers/mailer.rb#L18-L22 – may need to DRY this up in a separate initialiser (that sets it on the global `Rails.configuration` config) and then use that.